src/libpriv/rpmostree-unpacker.c

```diff
@@ -473,6 +477,17 @@ build_metadata_variant (RpmOstreeUnpacker *self,
     g_variant_builder_add (&metadata_builder, "{sv}", "rpmostree.metadata",
                            g_variant_new_from_bytes ((GVariantType*)"ay",
                                                      metadata, TRUE));
+
+    /* NB: the header also includes checksums for every entry in the (cut-off)
+     * archive, so there's no need to also bring that in */
```

Comment:
See also https://github.com/rpm-software-management/rpm/issues/163

We should probably start defining terms more carefully since "pkg_checksum" could be like 3-4 different things 😄 
How about `header_sha256` or something?